kolena/_experimental/object_detection/utils.py

```diff
@@ -77,15 +77,16 @@ def _compute_threshold_curve(
     if len(y_score) < 1:
         return None
 
-    thresholds = np.unique(y_score[y_score >= 0.0]).tolist()  # sorts
+    potential_thresholds = np.unique(y_score[y_score >= 0.0]).tolist()  # sorts
 
-    if len(thresholds) >= 501:
-        thresholds = list(np.linspace(min(thresholds), max(thresholds), 501))
+    if len(potential_thresholds) >= 501:
+        potential_thresholds = list(np.linspace(min(potential_thresholds), max(potential_thresholds), 501))
 
     precisions: List[float] = []
     recalls: List[float] = []
+    thresholds: List[float] = []
```

Comment:
It is better for the points of a PR curve can be found on the F1 curve (and vice versa)